docs/index.md

```diff
@@ -138,11 +152,18 @@ Available Commands:
 Flags:
       --base-dir.minimega string   base minimega directory (default "/tmp/minimega")
       --base-dir.phenix string     base phenix directory (default "/phenix")
+      --bridge-mode string         bridge naming mode for experiments ('auto' uses experiment name for bridge; 'manual' uses user-specified bridge name, or 'phenix' if not specified) (options: manual | auto)
+      --deploy-mode string         deploy mode for minimega VMs (options: all | no-headnode | only-headnode)
```

Comment:
default=no-headnode

Reply:
This was the output of `phenix help` as of commit `9a8496b` (Aug 5). I added the default option to the output manually, will update the CLI help message if I get time.